src/services/functionAppService.ts

```diff
@@ -226,6 +241,23 @@ export class FunctionAppService extends BaseService {
     return functionZipFile;
   }
 
+  public async updateFunctionAppSetting(functionApp: Site, setting: string, value: string) {
+    const baseUrl = `https://management.azure.com/subscriptions/${this.subscriptionId}` +
```

Comment:
Is there not a method in any of the imported SDK that we are using to use instead of manually crafting HTTP request?